onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -5540,6 +5540,14 @@ def aten_sym_size(self: TReal, dim: int = 0) -> TReal:
     """sym_size(Tensor self, int dim) -> Tensor"""
     # NOTE: onnxscript doesn't support attribute process,
     # so op.Shape(self, start=dim, end=dim + 1) is not supported.
+
+    # TODO(titaiwang): ORT==1.15 fixes SegFault
+    # https://github.com/microsoft/onnx-script/pull/484#discussion_r1136105039
+    # Change the op to:
+    # shape = op.Shape(self)
+    # idx= op.Reshape(dim, [1])
+    # return op.Gather(shape, idx)
```

Comment:
Why gather when we can slice? Just for my learning

Reply:
From Rama: As discussed in another PR, dim+1 also doesn't work if dim is -1. Hence, Gather seems a better solution.